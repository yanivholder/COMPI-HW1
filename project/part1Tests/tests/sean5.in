Int int "normal string with\nline break"