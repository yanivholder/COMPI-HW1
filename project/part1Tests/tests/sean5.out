1 ID Int
1 INT int
1 STRING normal string with
line break
